server/meca/file-generators/article.js

```diff
@@ -20,15 +23,14 @@ function createXmlObject(manuscript, editorsById, affiliations, reviewerMap) {
       'article-categories': {
         'subj-group': [].concat(
           {
-            '@subj-group-type': 'article-type',
-            subject: manuscript.meta.articleType,
-          },
-          {
-            '@subj-group-type': 'article-type-id',
+            '@subj-group-type': 'article_type',
             subject: articleTypeMap[manuscript.meta.articleType],
           },
           manuscript.meta.subjects.map(subject => ({
-            'subj-group': { '@subj-group-type': 'keywords', subject },
+            'subj-group': {
+              '@subj-group-type': 'subject_areas',
+              subject: subjectAreas[subject],
```

Comment:
It may be worth throwing an error if the subject does not exist within `subjectAreas`. At the moment, it will set the subject to undefined, and we would rely on EJP to return an error.

Reply:
True. But it's a very unlikely situation since the same list is used in the drop down which populates the database in the first place.